### PR TITLE
Created pages

### LINKED ISSUE #1: Create Render Jobs page
## Desktop
Desktop - Render Jobs
## Mobile
iPhone 8 - Render Jobs

### LINKED ISSUE #2: Create Render Jobs Details page
## Desktop
Desktop - Render Job Details
---
Info Tab
---
Statistics Tab
## Mobile
iPhone 8 - Render Job Details
---
Iphone 8 - Info Tab
---
Iphone 8 - Statistics Tab

### LINKED ISSUE #7: Add library for creating Charts
Add react library to create __charts__ in ___Material design__ style. 
## Examples:
Bar chart - horizontal
---
Bar chart - vertical
---
Line chart

### PR DESCRIPTION
**Pull request by ZaykaNya**
### Changes
* Created Render Jobs Page
* Created Render Job Details Page
* Added Charts library

### Issues
* Closes #1
* Closes #2
* Closes #7

### CODE REVIEW

File: .idea/runConfigurations/npm_start.xml [hunk 1]
```diff
@@ -6,6 +6,7 @@
       <script value="start" />
     </scripts>
     <node-interpreter value="project" />
+    <package-manager value="npm" />
```
Comment (DanilAndreev) [inline .idea/runConfigurations/npm_start.xml]: Change package-manager to yarn

File: package.json [hunk 2]
```diff
@@ -38,14 +38,21 @@
     "url": "https://github.com/DanilAndreev/pathfinder-monitor.git"
   },
   "dependencies": {
+    "@devexpress/dx-react-chart": "^2.7.2",
+    "@devexpress/dx-react-chart-material-ui": "^2.7.2",
+    "@devexpress/dx-react-core": "^2.7.2",
     "@material-ui/core": "^4.11.0",
     "@material-ui/icons": "^4.9.1",
+    "@types/react-window": "^1.8.2",
```
Comment (DanilAndreev) [inline package.json]: move this to dev dependencies

File: src/Components/DataTextField/DataTextField.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+               {label}
+           </Typography>
+           <Typography className={clsx(classes.boxContainerText, className)}>
+               {children}
+           </Typography>
+       </Box>
+   );
+});
+
+export default withStyles(styles)(DataTextField)
```
Comment (DanilAndreev) [inline src/Components/DataTextField/DataTextField.tsx]: line ending?

File: src/Components/LeftsideMenu/LeftsideMenu.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
+/*
```
Comment (DanilAndreev) [inline src/Components/LeftsideMenu/LeftsideMenu.tsx]: Delete this file

File: src/Components/Progress/Progress.tsx [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+            setProgress((prevProgress) => (prevProgress >= 100 ? 1 : prevProgress + 1));
+        }, 100);
+        return () => {
+            clearInterval(timer);
+        };
+    }, []);
+
+    return (
+        <div className={clsx(classes.root, className)}>
+            <LinearProgressWithLabel value={progress}/>
```
Comment (DanilAndreev) [inline src/Components/Progress/Progress.tsx]: Change div to Box

File: src/Components/SimpleList/SimpleList.tsx [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+const SimpleList = React.forwardRef((props: SimpleListProps, ref: Ref<any>) => {
+    const {
+        classes,
+        className,
+    } = props;
+
+    const {changeRoute} = useChangeRoute();
+
+    return (
+        <Box className={clsx(classes.root, className)}>
```
Comment (DanilAndreev) [inline src/Components/SimpleList/SimpleList.tsx]: Change to map

File: src/Components/TabsPanel/TabsPanel.tsx [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+    return (
+        <Box
+            role="tabpanel"
+            hidden={value !== index}
+            id={`full-width-tabpanel-${index}`}
+            aria-labelledby={`full-width-tab-${index}`}
+            {...other}
+        >
+            {value === index && (
```
Comment (DanilAndreev) [inline src/Components/TabsPanel/TabsPanel.tsx]: remove curly braces

File: src/Views/OrganizationPageView/LocalComponents/PluginComponent/PluginComponent.tsx [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+interface PluginComponentPropsStyled {
+    classes?: any;
+    style?: any;
+    className?: string;
+    plugin?: string;
+    description?: string;
+}
+
+const PluginComponent = React.forwardRef((props: PluginComponentPropsStyled, ref: Ref<any>) => {
```
Comment (DanilAndreev) [inline src/Views/OrganizationPageView/LocalComponents/PluginComponent/PluginComponent.tsx]: JSDoc

File: src/Views/OrganizationPageView/LocalComponents/PluginComponent/PluginComponent.tsx [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+ */
+
+import React, {Ref} from "react";
+import {withStyles, Box, Grid, Select, MenuItem, IconButton, ListItem} from "@material-ui/core";
+import styles from "./styles";
+import DataTextField from "../../../../Components/DataTextField";
+import clsx from "clsx";
+import CloseIcon from "@material-ui/icons/Close";
+
+interface PluginComponentPropsStyled {
```
Comment (DanilAndreev) [inline src/Views/OrganizationPageView/LocalComponents/PluginComponent/PluginComponent.tsx]: extend Stylable

File: src/Views/OrganizationPageView/LocalComponents/TopicWithButton/TopicWithButton.tsx [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+
+import React, {Ref} from "react";
+import {IconButton, withStyles} from "@material-ui/core";
+import {Avatar, Grid, Box, Typography, Divider} from "@material-ui/core";
+import clsx from "clsx";
+import styles from "./styles";
+import AddIcon from '@material-ui/icons/Add';
+
+
+interface TopicWithButtonPropsStyled{
```
Comment (DanilAndreev) [inline src/Views/OrganizationPageView/LocalComponents/TopicWithButton/TopicWithButton.tsx]: JSDoc

File: src/Views/OrganizationPageView/LocalComponents/TopicWithButton/TopicWithButton.tsx [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+    classes?: any;
+    style?: any;
+    className?: string;
+    children?:string,
+}
+
+const TopicWithButton = React.forwardRef((props:TopicWithButtonPropsStyled)=>{
```
Comment (DanilAndreev) [inline src/Views/OrganizationPageView/LocalComponents/TopicWithButton/TopicWithButton.tsx]: JSDoc

File: src/Views/OrganizationPageView/OrganizationPageView.tsx [hunk 12]
```diff
@@ -0,0 +1,166 @@
...
+import styles from "./styles";
+import DataTextField from "../../Components/DataTextField";
+import clsx from "clsx";
+import TopicWithButton from "./LocalComponents/TopicWithButton";
+import BuildIcon from '@material-ui/icons/Build';
+import SettingsIcon from '@material-ui/icons/Settings';
+import PluginComponent from "./LocalComponents/PluginComponent";
+
+
+interface OrganizationPageViewPropsStyled {
```
Comment (DanilAndreev) [inline src/Views/OrganizationPageView/OrganizationPageView.tsx]: JSDoc

File: src/Views/OrganizationPageView/OrganizationPageView.tsx [hunk 13]
```diff
@@ -0,0 +1,166 @@
...
+    classes?: any;
+    style?: any;
+    className?: string;
+
+}
+
+const OrganizationPageView = React.forwardRef((props: OrganizationPageViewPropsStyled, ref: Ref<any>) => {
```
Comment (DanilAndreev) [inline src/Views/OrganizationPageView/OrganizationPageView.tsx]: JSDoc

File: src/Views/RenderJobsDetailsView/Tabs/TasksTab/styles.ts [hunk 14]
```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2020. This code created and belongs to Pathfinder render manager project.
+ * Owner and project architect: Danil Andreev | danssg08@gmail.com |  https://github.com/DanilAndreev
+ * File creator: Andrii Demchyshyn
+ * Project: pathfinder-monitor
+ * File last modified: 01.10.2020, 18:22
+ * All rights reserved.
+ */
+
```
Comment (DanilAndreev) [inline src/Views/RenderJobsDetailsView/Tabs/TasksTab/styles.ts]: remove duplicate copyright

File: src/Views/UserPageView/LocalComponents/OrganizationsFieldsRow/OrganizationsFieldsRow.tsx [hunk 15]
```diff
@@ -0,0 +1,50 @@
...
+ */
+
+import React from "react";
+import {Grid, withStyles} from "@material-ui/core";
+
+import styles from "../../styles";
+import clsx from "clsx";
+import DataTextField from "../../../../Components/DataTextField";
+
+interface OrganizationsFieldsRowPropsStyled{
```
Comment (DanilAndreev) [inline src/Views/UserPageView/LocalComponents/OrganizationsFieldsRow/OrganizationsFieldsRow.tsx]: extend Stylable and add JSDoc

File: src/Views/UserPageView/LocalComponents/OrganizationsFieldsRow/OrganizationsFieldsRow.tsx [hunk 16]
```diff
@@ -0,0 +1,50 @@
...
+    classes?: any,
+    style?:any,
+    className?: string,
+    organization?: string,
+    role?: string,
+    status?: string,
+}
+
+const OrganizationsFieldsRow = React.forwardRef((props:OrganizationsFieldsRowPropsStyled) =>{
```
Comment (DanilAndreev) [inline src/Views/UserPageView/LocalComponents/OrganizationsFieldsRow/OrganizationsFieldsRow.tsx]: JSDoc

File: src/Views/UserPageView/LocalComponents/TokensViewer/TokensViewer.tsx [hunk 17]
```diff
@@ -0,0 +1,77 @@
...
+    Box
+} from "@material-ui/core";
+import clsx from "clsx";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+
+
+import styles from "./styles";
+import DataTextField from "../../../../Components/DataTextField/DataTextField";
+
+interface TokensViewerPropsStyled {
```
Comment (DanilAndreev) [inline src/Views/UserPageView/LocalComponents/TokensViewer/TokensViewer.tsx]: extend Stylable and add JSDoc

File: src/Views/UserPageView/LocalComponents/TokensViewer/TokensViewer.tsx [hunk 18]
```diff
@@ -0,0 +1,77 @@
...
+    classes?: any,
+    style?: any,
+    className?: string,
+}
+
+const TokensViewer = React.forwardRef((props: TokensViewerPropsStyled) => {
```
Comment (DanilAndreev) [inline src/Views/UserPageView/LocalComponents/TokensViewer/TokensViewer.tsx]: JSDoc

File: src/Views/UserPageView/UserPageView.tsx [hunk 19]
```diff
@@ -0,0 +1,58 @@
...
+import styles from "./styles";
+import {Avatar, Grid, Box, Typography, Divider} from "@material-ui/core";
+import githubAvatar from "./githubAvatar.jpg";
+import DataTextField from "../../Components/DataTextField";
+import OrganizationsFieldsRow from "./LocalComponents/OrganizationsFieldsRow";
+import clsx from "clsx";
+import TokensViewer from "./LocalComponents/TokensViewer";
+
+
+interface UserPageViewPropsStyled {
```
Comment (DanilAndreev) [inline src/Views/UserPageView/UserPageView.tsx]: extend Stylable and add JSDoc

File: src/Views/UserPageView/UserPageView.tsx [hunk 20]
```diff
@@ -0,0 +1,58 @@
...
+    classes?: any;
+    style?: any;
+    className?: string;
+}
+
+const UserPageView = React.forwardRef((props: UserPageViewPropsStyled, ref: Ref<any>) => {
```
Comment (DanilAndreev) [inline src/Views/UserPageView/UserPageView.tsx]: JSDoc

**Review by DanilAndreev — CHANGES_REQUESTED** [review]

File: src/Components/SimpleList/SimpleList.tsx (hunk 6 above)
Comment (ZaykaNya) [reply src/Components/SimpleList/SimpleList.tsx]: Fixed

**Review by DanilAndreev — APPROVED** [review]

**Pull request merged**